### ISSUE TITLE
Support for elm-css stylesheets

### ISSUE THREAD

**Issue by MazeChaZer** [halfzebra/create-elm-app#134]
@halfzebra What do you think about this feature?
If you like to support this, I would sit down over the weekend and take a shot at this.

I assume one problem would be to determine which Elm file should be compiled into a CSS stylesheet (perhaps always the file `src/Styles.elm`? Another subdirectory? ...)

**Comment by halfzebra** [comment]
Hello @MazeChaZer!

Thank you for the contribution!

This is a hot topic at the moment for Create Elm App and we are pretty open to the idea of adding **optional** elm-css support. You are very welcome to work on a PR 👍 

How would you go about implementing it?

**Comment by MazeChaZer** [comment]
I didn't do much research yet, but I thought that I could just add [elm-css-webpack-loader](https://github.com/tcoopman/elm-css-webpack-loader) to the webpack config.
Something along the lines of
```js
{
    test: /css\.elm$/,
    use: [
        'style-loader',
        'css-loader',
        'elm-css-webpack-loader'
    ]
}
```
AFAIK this rule would compile all `.elm` files within the `css/` directory into stylesheets (example based on #124).

**Comment by halfzebra** [comment]
I don't have a solid opinion on the code organization, but I'd like to:

- Get the CSS from `elm-css` into the main CSS bundle during the build.
- Use `noParse`, check out [this PR](https://github.com/tcoopman/elm-css-webpack-loader/pull/16/commits/de53eb7cc4440a42261a9cc78ce5ffcfb9d584ac) for more details.

 We can probably start with `Stylesheet.elm` and then consider using a separate folder if anyone is interested.

**Comment by MazeChaZer** [comment]
@halfzebra Sounds good :+1:

**Comment by halfzebra** [comment]
I have merged the docs from #137 and they are available in [User Guide](https://github.com/halfzebra/create-elm-app/blob/master/template/README.md#using-elm-css)

@MazeChaZer thanks for your help! 👍
